packages/development-proxy/mixin.core.js

```diff
@@ -26,14 +24,6 @@ class ProxyMixin extends Mixin {
     };
 
     if (typeof proxyConfig === 'string') {
-      if (!checkUrl(proxyConfig)) {
-        console.warn(
-          `The proxy target '${proxyConfig}' is invalid, disabling feature`
-        );
-
-        return;
```

Comment:
I think we still need to return here as this allows to have a setup like this:
```js
{
  proxy: {
    '/my/api': 'PROXY_URL'
  }
}
```

Otherwise you would have to either always define the PROXY_URL or to remove the config entry when you want to start without a proxy.